### ISSUE TITLE
Compatibility with current version of Beanie (1.17)

### ISSUE THREAD

**Issue by bkis** [fastapi-users/fastapi-users-db-beanie#9]
## Describe the bug

As [Beanie v1.17 was released on Jan 19](https://github.com/roman-right/beanie/releases/tag/1.17.0) and `fastapi-users-db-beanie` has it's dependencies defined as

```toml
dependencies = [
    "fastapi-users >= 10.0.1",
    "beanie >=1.11.0,<1.17",
]
```

users that are on the current Beanie version (1.17) who want to install `fastapi-users[beanie]` will not be able to.  
In short: Beanie dependency version should be bumped again (after testing compatibility, of course).

## To Reproduce

1. Have a project using Beanie 1.17
2. Try to install `fastapi-users[beanie]`

## Expected behavior
`fastapi-users[beanie]` should be compatible with the current version of Beanie.

## Configuration
 - Python version : `3.10`
 - FastAPI version : `0.89`
 - FastAPI Users version : `1.1.3`

**Comment by bkis** [comment]
You're quite fast to respond :+1:

**Comment by frankie567** [comment]
😄⚡️

**Comment by frankie567** [comment]
Released in [`v1.1.4`](https://github.com/fastapi-users/fastapi-users-db-beanie/releases/tag/v1.1.4)
